ontology/bmont.md

```diff
@@ -0,0 +1,18 @@
+---
+id: bmont
+      mirror_from: https://raw.githubusercontent.com/SCAI-BIO/BiomarkerOntology/main/BMONT-merged.owl
+      title: "Biomarker Ontology"
+      contact:
+        email: alpha.tom.kodamullil@scai.fraunhofer.de
+        label: Alpha Tom Kodamullil
+        orcid: 0000-0001-9896-3531
+        github: akodamullil
+      description: The Biomarker Ontolody comprises a comprehensive knowledge involving a variety of fields of medical and biological aspects.
```

Comment:
The description for the ontology can be improved to better reflect the nature and scope of BMONT. Since this is an application ontology, I would suggest something like so:

"An application ontology that represents biomarkers..."

Please see https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1968 for recommendations.